bin/makisu/cmd/diff.go

```diff
@@ -94,11 +99,36 @@ func (cmd *diffCmd) Diff(imagesFullName []string) error {
 				panic(fmt.Errorf("untar image %d layer reader: %s", i+1, err))
 			}
 		}
+
 		memFSArr = append(memFSArr, memfs)
+		reader, err := store.Layers.GetStoreFileReader(manifest.GetConfigDigest().Hex())
+		if err != nil {
+			panic(fmt.Errorf("get image%d config file reader %s: %s", i+1, manifest.GetConfigDigest().Hex(), err))
+		}
+
+		configBytes, err := ioutil.ReadAll(reader)
+		if err != nil {
+			panic(fmt.Errorf("read image%d config file %s: %s", i+1, manifest.GetConfigDigest().Hex(), err))
+		}
+
+		config := new(image.Config)
+		if err := json.Unmarshal(configBytes, config); err != nil {
+			panic(fmt.Errorf("unmarshal image%d config file %s: %s", i+1, manifest.GetConfigDigest().Hex(), err))
+		}
+		imageConfigs = append(imageConfigs, config)
+	}
+
+	log.Infof("* compare image %s and image %s config", pullImages[0].GetRepository()+":"+pullImages[0].GetTag(), pullImages[1].GetRepository()+":"+pullImages[1].GetTag())
+	if configDiff := cmp.Diff(imageConfigs[0], imageConfigs[1], cmpopts.IgnoreUnexported(image.Config{})); configDiff != "" {
+		// Format the diff string.
+		configDiff = strings.ReplaceAll(configDiff, "\n", "")
```

Comment:
Maybe we should stop using json logger? then there might not be need for these replacements.

Reply:
the configDiff itself has many weird `\n \t` characters... so I think it kind of necessary to format it.

Reply:
If we don't print if as part of a json log line it should look better.